README.md

```diff
@@ -216,7 +216,23 @@ and `{group}` are unix user and group names the server will run under.
     'Client secret'.
 
 
-## 4. Mandrill transactional e-mail servis
+## 4. Google custom search engine key
+
+ 1. Sign into [Control Panel](https://cse.google.com/create/new) using your Google Account.
+
+ 2. In the **Sites to search** section, add the `www.chcemvediet.sk` page. Set **Language** to
+    `Slovak`and click **Create**. 
+
+ 3. Go to the **Look and feel** section and click the **Layout** tab. Select and save the `Results
+    only` layout.
+
+ 4. Go back to **Setup** section. On the **Basics** tab, in the **Details** section, copy **Search
+    engine ID**.
+
+ 5. Run `setup.py` to configure your google custom search API key.
```

Comment:
Vsimol som si, ze v README nam ostal na niektorych miestach stary nazov setup scriptu `configure.py`. Pls oprav to vsade na `setup.py`

Reply:
:ok: